test/bindings/http/promiscuous_receiver_test.js

```diff
@@ -67,17 +74,53 @@ describe("HTTP Transport Binding Receiver for CloudEvents", () => {
 
     it("Binary data returns a CloudEvent", () => {
       const headers = {
-        [constants.HEADER_CONTENT_TYPE]: constants.DEFAULT_CONTENT_TYPE,
-        [constants.DEFAULT_SPEC_VERSION_HEADER]: specversion,
-        [constants.BINARY_HEADERS_03.ID]: id,
-        [constants.BINARY_HEADERS_03.TYPE]: type,
-        [constants.BINARY_HEADERS_03.SOURCE]: source
+        [HEADER_CONTENT_TYPE]: DEFAULT_CONTENT_TYPE,
+        [DEFAULT_SPEC_VERSION_HEADER]: specversion,
+        [BINARY_HEADERS_03.ID]: id,
+        [BINARY_HEADERS_03.TYPE]: type,
+        [BINARY_HEADERS_03.SOURCE]: source
       };
 
       const event = receiver.accept(headers, data);
       validateEvent(event, specversion);
     });
   });
+
+  describe("Kafka-Knative event source", () => {
+    const specversion = "1.0";
+    const id = "partition:1/offset:23";
+    const type = "dev.knative.kafka.event";
+    const source =
+      "/apis/v1/namespaces/kafka/kafkasources/kafka-source#knative-demo-topic";
+
+    it("Should be parseable", () => {
```

Comment:
Minor typo `parsable` instead of `parseable`.

Reply:
Gotcha. Fixed.